tests-invoke

```diff
@@ -252,8 +256,21 @@ class PullTask(object):
             cmd = ["timeout", "120m", entry_point]
             if opts.verbose:
                 cmd.append("--verbose")
-            ret = subprocess.call(cmd)
-
+            # Popen instead of call; while not etc... (SubProcess.poll)
```

Comment:
Please drop this comment. Or better even, say "# cancel test if PR gets updated while test is running"